packages/block-library/src/social-link/README.md

```diff
@@ -0,0 +1,21 @@
+# Social Icon block
+
+The Social Icon is a foundational block that displays various icons linking to different social profiles or sites. Each social service is registered as a variation of this block.
+
+
+## Adding new social icons
+
+The core maintains certain standards for social icons. To add a new one to the existing variations, it must be a well-established, popular, and mainstream service.
+
+When considering establishment and popularity, several factors should be evaluated, such as:
+
+* Is the service popular enough for core developers to have heard of it before? Is it "mainstream?"
+* How old is the service?
+* Does it have a well-established Wikipedia article? (Seriously.)
```

Comment:
"Does it have a Wikipedia entry?" 

I don't think you can gauge Wikipedia articles on 'well-established'. I think it's binary; either the service has one or it doesn't.